mountpoint-s3/tests/fuse_tests/fork_test.rs

```diff
@@ -380,6 +384,130 @@ fn mount_scoped_credentials() -> Result<(), Box<dyn std::error::Error>> {
     Ok(())
 }
 
+#[cfg(all(feature = "sse_kms", not(feature = "s3express_tests")))]
+fn mount_with_sse(
+    bucket: &str,
+    mount_point: &Path,
+    prefix: &str,
+    key_id: &str,
+    credentials: Option<aws_sdk_s3::config::Credentials>,
+) -> Child {
+    let region = get_test_region();
+    let mut cmd = Command::cargo_bin("mount-s3").expect("can not locate mount-s3 binary");
+    cmd.stdout(Stdio::piped())
+        .arg(bucket)
+        .arg(mount_point)
+        .arg(format!("--region={region}"))
+        .arg(format!("--prefix={prefix}"))
+        .arg("--sse=aws:kms:dsse")
+        .arg(format!("--sse-kms-key-id={key_id}"))
+        .arg("--auto-unmount")
+        .arg("--foreground");
+    if let Some(credentials) = credentials {
+        cmd.env("AWS_ACCESS_KEY_ID", credentials.access_key_id())
+            .env("AWS_SECRET_ACCESS_KEY", credentials.secret_access_key())
+            .env("AWS_SESSION_TOKEN", credentials.session_token().unwrap());
```

Comment:
should we assume the session token is always set?

i'd recommend we just check if its set and optionally apply it.

https://github.com/awslabs/mountpoint-s3/blob/16cb2ed1baf218083aa48e96c1059c53e6983680/mountpoint-s3/tests/auth-cli.rs#L34-L36